### PR TITLE
Showing default (expected) values for high/low kdump memory.

### PR DESCRIPTION
**Pull request by schubi2**
**Problem**
The user does not know the default kdump memory values which have been given by the system.
**Solution**
Showing the user the default values too.

### CODE REVIEW

File: src/include/kdump/dialogs.rb [hunk 1]
```diff
@@ -523,14 +523,19 @@ def tabs
...
+      low_min = Kdump.memory_limits[:min_low].to_i
+      low_max = Kdump.memory_limits[:max_low].to_i
+      low_default = Kdump.memory_limits[:default_low].to_i
       low_label = if Kdump.high_memory_supported?
         _("Kdump &Low Memory [MiB]")
       else
         _("Kdump Memor&y [MiB]")
       end
-      low_label += " (#{min} - #{max})"
+      # TRANSLATORS: Each momory value is based on MiB
```
Comment (jreidinger) [inline src/include/kdump/dialogs.rb]: typo momory and allso it is good that you mention it for translators, but it is not mentioned even in english not? It is already in label above, so I think it does not bring much value for translators.

Comment (jreidinger) [reply src/include/kdump/dialogs.rb]: better mention to not modify that `%{}` part.

File: src/include/kdump/dialogs.rb [hunk 2]
```diff
@@ -539,37 +544,44 @@ def kdump_memory_widget
...
+        Left(Label(low_range))
       ]
       if Kdump.high_memory_supported?
-        min = Kdump.memory_limits[:min_high].to_i
-        max = Kdump.memory_limits[:max_high].to_i
+        high_min = Kdump.memory_limits[:min_high].to_i
+        high_max = Kdump.memory_limits[:max_high].to_i
+        high_default = Kdump.memory_limits[:default_high].to_i
+        # TRANSLATORS: Each momory value is based on MiB
+        high_range = format(_("(%{high_min} - %{high_max}, default: %{high_default})"),
```
Comment (jreidinger) [inline src/include/kdump/dialogs.rb]: do we really need new translatable string for it? Why not use `_("(%{min} - %{max}, default: %{default})"` and have it consistently translated?

**Comment by teclator** [comment]
NP: as this is a UI change, usually adding a screenshot section with the previous interface and the current one help to see the change. I know it is in trello, but trello cards are usually archived. Also as part of the problem, at the end we usually link the trello card and the bugzilla link.

**Pull request merged**